src/routes/jira/workspaces/jira-workspaces-get.ts

```diff
@@ -1,3 +1,56 @@
-export const JiraWorkspacesGet = () => {
-	// Content included in ARC-2115-search-connected-orgs
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { RepoSyncState } from "models/reposyncstate";
+import { Errors } from "config/errors";
+import { paginatedResponse } from "utils/paginate-response";
+
+export type Workspace = {
+	id: string,
+	name: string,
+	canCreateContainer: boolean
+}
+
+const DEFAULT_PAGE_NUMBER = 1; // Current page
+export const DEFAULT_LIMIT = 20; // Number of items per page
+
+const findMatchingOrgs = async (subscriptions: Subscription[], orgName?: string): Promise<Workspace[]> => {
+	const matchingRepos = await Promise.all(subscriptions.map(async (subscription: Subscription) => {
+		return orgName ?
+			await RepoSyncState.findByOrgNameAndSubscriptionId(subscription, orgName) :
+			await RepoSyncState.findOneFromSubscription(subscription);
+	}));
+
+	const matchedOrgs = matchingRepos
+		.filter((org): org is RepoSyncState => org !== null)
```

Comment:
/nit
Nothing to do, this is only a reminder to me how does this `org is RepoSyncState =>` work. Never saw it before. Interesting (and nice 👍 )